xbmc/utils/URIUtils.h

```diff
@@ -100,12 +100,54 @@ class URIUtils
    */
   static std::string GetDiscBasePath(const std::string& file);
 
+  /*! \brief Given a bluray:// path, return the underlying file path (eg. smb://, udf:// etc..)
+   \param url CURL containing bluray:// path.
+   \return return the underlying file path.
+   */
+  static std::string GetDiscUnderlyingFile(const CURL& url);
```

Comment:
> Given a bluray:// path

To reflect that this method requires a bluray path, I would name it respectively, `GetBlurayDiscUnderlyingFile`